src/parser/statement.js

```diff
@@ -844,7 +844,7 @@ pp.parseExport = function (node) {
     }
     node.declaration = expr;
     if (needsSemi) this.semicolon();
-    this.checkExport(node);
+    this.checkExport(node, true, true);
```

Comment:
Yeah, was trying to follow the style I saw elsewhere, but agree. Would you like me to change that to an object?